catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -282,6 +282,7 @@
 
     <feature name="catalog-security-filter" version="${project.version}"
              description="Catalog Security Filtering">
+        <feature prerequisite="true">catalog-core-api</feature>
```

Comment:
❓ I know @tbatie would ask this if he were reviewing it before me 😄 : Is the `prerequisite="true"` necessary? Or will the `catalog-security-filter` bundle be able to start up and wait until `catalog-core-api` is finished starting? It's a smell that bundles are relying on start/refresh order.

Reply:
`catalog-security-filter` has import dependencies on `catalog-core-api` so...

Reply:
I could be wrong, but I don't think the prerequisite is needed for something like package imports with the way Karaf's Feature Resolver works. I would need to test it out to be sure though, so I'm going to approve as is.

Reply:
I would think just having the feature and the import dependencies should be enough for it to wait. Prerequisite stops the resolution process and then restarts it after its active so it can make startup take a lot longer among other issues. I think we should only use Prerequisite when OSGi doesn't have any way to determine the start order through imports or otherwise(which we should try to avoid).